.github/remove_package.sh

```diff
@@ -34,4 +34,9 @@ echo "${GH_BODY}" | while read url ; do
     jq 'del(.[] | select((.|ascii_downcase) == ("'$url'"|ascii_downcase)))' packages.json > temp.json
     mv temp.json packages.json
     echo "- '$url'."
+
+    # 1e. Add Item to Denylist
+    jq '. |= . + [{"package_url": "'$url'", "notes": "Requested in https://github.com/SwiftPackageIndex/PackageList/issues/'$GH_ISSUE'."}]' -S denylist.json > temp.json
```

Comment:
If `jq` wants the nodes in alpha order, should we flip them here too? I guess it'll fix it up as it runs so it probably doesn't matter 😂

Reply:
It doesn't make a difference, but I'll switch them for you!

Reply:
Let's keep `jq` employed by having something to do 😂